Lib/unittest/case.py

```diff
@@ -617,6 +616,9 @@ def run(self, result=None):
                 _addSkip(result, self, skip_why)
                 return result
 
+            # Increase the number of running tests only if it hasn't been skipped
```

Comment:
it's not "running tests", it's "tests run". Also, line length > 80


```suggestion
            # Increase the number of tests only if it hasn't been skipped
```